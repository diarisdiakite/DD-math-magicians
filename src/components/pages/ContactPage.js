import React, { useEffect } from 'react';
import { useNavigate } from 'react-router-dom';

function ContactPage() {
  const navigate = useNavigate();

  useEffect(() => {
    setTimeout(() => {
      navigate(-1);
    }, 10000);
  });

  return (
    <div>
      <h2>This Page is not found</h2>
      <p>
        Please contact us at
        <strong> diarisdiakite@gmail.com</strong>
      </p>
    </div>
  );
}

export default ContactPage;
