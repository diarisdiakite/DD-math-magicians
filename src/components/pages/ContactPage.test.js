import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import ContactPage from './ContactPage';

describe('ContactPage component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <ContactPage />
        </MemoryRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
