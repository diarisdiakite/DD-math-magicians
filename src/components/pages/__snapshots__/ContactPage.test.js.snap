// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<div
  className="main-container"
>
  <h2>
    This Page is not found
  </h2>
  <p>
    Please contact us at
    <strong>
       diarisdiakite@gmail.com
    </strong>
  </p>
</div>
`;
